packages/jest-resolve/src/__tests__/resolve.test.ts

```diff
@@ -345,6 +345,53 @@ describe('findNodeModule', () => {
       );
     });
 
+    test('supports nested pattern', () => {
+      const result = Resolver.findNodeModule('#nested', {
+        basedir: path.resolve(importsRoot, './nested-import/index.cjs'),
+        conditions: ['node', 'require'],
+      });
+
+      expect(result).toEqual(
+        path.resolve(importsRoot, './nested-import/node.cjs'),
+      );
+    });
+
+    describe('array pattern', () => {
+      test('supports array pattern - resolve to first found', () => {
```

Comment:
remove `supports array pattern - ` from the nested tests

Reply:
👍